custom_components/easee/const.py

```diff
@@ -326,6 +359,111 @@
         ],
         "units": None,
         "convert_units_func": None,
+        "device_class": None,
         "icon": "mdi:currency-usd",
     },
+    "eq_status": {
+        "type": "eq_sensor",
+        "key": "state.isOnline",
+        "attrs": [
+            "state.latestPulse",
+            "state.clockAndDateMeter",
+            "state.rcpi",
+            "state.localRSSI",
+            "state.softwareRelease",
+            "state.latestFirmware",
+        ],
+        "units": None,
+        "convert_units_func": None,
+        "device_class": DEVICE_CLASS_SIGNAL_STRENGTH,
```

Comment:
DEVICE_CLASS_CONNECTIVITY

Reply:
I do not think that is possible on a sensor, only binary_sensor?

Reply:
I missed that it was a sensor.